.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,63 @@
+name: Bug report
+description: Submit a bug report
+labels: ["type-bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          # New to Python?
+
+          The issue tracker isn't the right place to get help.
+          Consider instead:
+
+            - reading the [Python tutorial](https://docs.python.org/3/tutorial/)
+            - posting on [Discourse](https://discuss.python.org/c/users/7)
+            - emailing [python-list](https://mail.python.org/mailman/listinfo/python-list)
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      description: Bugs in third-party projects (e.g. `requests`) do not belong in the CPython issue tracker
+      options:
+        - label: I am confident this is a bug in CPython, not a bug in a third-party project
+          required: true
+        - label: |
+            I have searched the [CPython issue tracker](https://github.com/python/cpython/issues),
+            and am confident this bug has not been reported before
+          required: true
+  - type: textarea
+    attributes:
+      label: A clear and concise description of the bug
+      description: |
+        Tell us what happened.
+        Include a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) if possible.
+        Put any code blocks inside triple backticks.
+
+      value: |
+        ```py
+        your code here
+        ```
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: CPython versions tested on
+      multiple: true
+      options:
+        - "3.8"
+        - "3.9"
+        - "3.10"
+        - "3.11"
+        - "3.12"
+        - "CPython main branch"
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Operating systems and architectures tested on
```

Comment:
I considered both `dropdown` and `input`. I decided against `dropdown` because there are a lot of different system/architecture combinations you could have, so it might end up being a long dropdown menu (and it's unlikely we could think of all possible options between us, anyhow). I decided against `input` because the person reporting the bug might have tested on multiple operating systems, in which case they'll probably want to present this information in the form of a bulleted list.

Reply:
I was thinking that often the OS/architecture is not relevant to the issue, and if they report it, it's just the one they happen to be using (usually linux/win/mac).  A dropdown makes it easier for the common case since they only need a couple of clicks and no typing, whereas people with more complex/specific combinations are better off describing it in the bug description textarea.  This could also be made explicit by adding a line like "If your OS/architecture is missing, please specify it in the next section".

Reply:
> I was thinking that often the OS/architecture is not relevant to the issue

Agreed, this is why I didn't make it a "required" input box. Maybe we could change the title to "Operating system and architecture (if relevant)", to make it clearer that you don't have to bother with this if it obviously isn't platform-specific. Do you think that would be enough?

Reply:
How's it look after the last push?

Reply:
It's still bigger than all the titles of all the other sections below.  Actually I'm not even sure the disclaimer needs a title :thinking:

Reply:
I'm a bit concerned that with two textareas people might end up writing the bug description in the first one, but I guess we can try and see if the proposed version works or not.

Reply:
Okay, I've tried something new in https://github.com/python/cpython/pull/107920/commits/79fb56e43ea81c6eea5f1343c2b5441ba0694947 -- what do you think now?